subsys/net/lib/lwm2m/Kconfig

```diff
@@ -183,6 +183,13 @@ config LWM2M_FIRMWARE_PORT_SECURE
 	help
 	  This is the default server port to connect to for LwM2M firmware downloads over coaps.
 
+config LWM2M_SERVER_OBJECT_VERSION_1_1
```

Comment:
I'd rather see this as a choice parameter, it sounds more natural for this kind of configs (in case we have more to choose than only 1.0 and 1.1), something like:
```
choice
	prompt "LwM2M Server object version"
	default LWM2M_SERVER_OBJECT_VERSION_1_0
# in the future we could add default LWM2M_SERVER_OBJECT_VERSION_1_1 if LWM2M_VERSION_1_1
	help
	  Select which version of the Server object should be used.

config LWM2M_SERVER_OBJECT_VERSION_1_0
	bool "Server object version 1.0"

config LWM2M_SERVER_OBJECT_VERSION_1_1
	bool "Server object version 1.1"

endchoice
```

Reply:
Ah Looks good. I will change this.